website/docs/tutorials/managing-access.md

```diff
@@ -44,6 +44,17 @@ A Resource's or Container's ACL is only accessible to your app if the following
 Access can be granted to individual [agents](../glossary#agent), to groups, or even to everyone.
 We currently only support reading what access has been granted to individual agents specifically.
 
+### Fetching access information
+
+There are two main ways of checking the access information of a Resource:
+
+1. Fetch the entire Resource, data and information at once. To do so, use the functions
+   `unstable_fetchFile` or `unstable_fetchLitDatasetWithAcl`. The returned value includes both the Resource
+   data (e.g. your profile or friend list), the `ResourceInfo`, and the ACL containing the associated
+   access information.
+2. Fetch only the information about a Resource, without fetching the Resource itself
+   with `fetchResourceInfo`, and then fetching the associated ACL (if any).
```

Comment:
Also, we don't currently expose a function to explicitly fetch a Resource's ACL (because we're pretending it's part of the Resource). So maybe what we need to satisfy Pod Manager's use case, is an `unstable_fetchResourceInfoWithAcl` that does the same as `unstable_fetchLitDatasetWithAcl`, but without fetching the contents of the Resource itself.

(I also wonder how esoteric their use case is - i.e. do we even need to mention it in this tutorial? Maybe we should add some point in the future add an "Advanced" section that has tutorials such as "Traversing Containers in a Pod"?)

Reply:
100% agree on the need for `unstable_fetchResourceInfoWithAcl`. And good point, I don't expect this to be the most common use case, so it may be better not to mention it at all here.

Reply:
Fixed in b18586db675fc622a332bc4cb576712a0d675d75